test/e2e_env/universal/auth/dp_auth.go

```diff
@@ -42,9 +41,10 @@ networking:
 		Expect(err).ToNot(HaveOccurred())
 
 		// then
-		Eventually(func() (string, error) {
-			return env.Cluster.GetKuma().GetKumaCPLogs()
-		}, "30s", "1s").Should(ContainSubstring("you are trying to override existing dataplane to which you don't have an access"))
+		// todo(jakubdyszkiewicz) uncomment once we can handle CP logs across all parallel executions
```

Comment:
This is for the next PR?

Reply:
yes, that's a separate PR.

from the description
```
GetCPLogs does not work with parallelized env. This should be fine for now, it's used only a couple of times. We can fix it in the next PRs, because we need to fix logs for Universal setups anyways.
```